tests/baselines/reference/convertKeywordsYes.errors.txt

```diff
@@ -0,0 +1,383 @@
+tests/cases/compiler/convertKeywordsYes.ts(292,10): error TS2300: Duplicate identifier '(Missing)'.

```

Comment:
This is likely a bug in the binder.  It should not be reporting collisions for declarations that we don't have a name for.